buildConfiguration.xml

```diff
@@ -0,0 +1,63 @@
+<root>
+  <dotnetArchitecture>x64</dotnetArchitecture>
+  <dotnetChannel>preview</dotnetChannel>
+  <dotnetVersion>1.0.0-preview2-002911</dotnetVersion>
+  <dotnetUrl>https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1</dotnetUrl>
+  <coreFxOldVersion>-rc3-\*</coreFxOldVersion>
+  <coreFxNewVersion>-rc3-24128-00</coreFxNewVersion>
+  <netCoreAppOldVersion>1.0.0-rc3-\*</netCoreAppOldVersion>
+  <netCoreAppNewVersion>1.0.0-rc3-004338</netCoreAppNewVersion>
+  <nugetVersion>3.5.0-rc-1285</nugetVersion>
+  <projects>
+    <src>
+      <project name="Microsoft.IdentityModel.Logging">
+        <build>yes</build>
+      </project>
+      <project name="Microsoft.IdentityModel.Tokens">
+        <build>yes</build>
+      </project>
+      <project name="System.IdentityModel.Tokens.Jwt">
+        <build>yes</build>
+        <dependency>Microsoft.IdentityModel.Tokens</dependency>
+      </project>
+      <project name="System.IdentityModel.Tokens.Saml">
+        <build>yes</build>
```

Comment:
Let's take it out until we actually use it. If you want to leave it in but commented out as a reminder, that's fine, but otherwise it's confusing.